make_prg/__main__.py

```diff
@@ -38,15 +38,6 @@ def setup_common_last_options(parsers: List[argparse.ArgumentParser]):
             help="Number of threads. 0 will use all available. Default: %(default)d",
         )
 
-        par.add_argument(
-            "-g",
-            "--output-graphs",
-            dest="output_graphs",
-            action="store_true",
-            default=False,
-            help="Outputs the recursive tree graphical representation (for development use only)"
-        )
-
```

Comment:
You could just make this a hidden arg? https://stackoverflow.com/questions/11114589/creating-hidden-arguments-with-python-argparse
Saves having to do all the nother stuff associated with this commit? Although seems like it'll do the same thing, so leave it as is as undoing this commit will create more work

Reply:
Yeah, I didn't know that. I think it can be implemented with both hidden args and env variables. Env variable makes some stuff easier, like [this method](https://github.com/leoisl/make_prg/blob/update_PRs_2/make_prg/utils/misc.py#L22-L23) to check if we should output debug graphs, that is used in some places in the code without needing to pass the argument over and over. I think we can keep like this for now, and maybe create an issue to refactor it?

Reply:
I think just leave it as it is. As long as we (you) know how to generate the trees that's all that matters. Don't worry about an issue for it